src/components/security-analysis-result.js

```diff
@@ -180,9 +180,17 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
         function sortAndAddResults(result, array) {
             const compareValue = (a, b) => {
                 const mult = reverse ? 1 : -1;
-                if (a === undefined && b === undefined) return 0;
-                else if (b === undefined) return -mult;
-                else if (a === undefined) return mult;
+                if (a === undefined && b === undefined) {
+                    return 0;
+                } else {
```

Comment:
Here, in addition to using more vertical space, it uses more horizontal space :grin:
-> `else if` is not forbidden

Reply:
Space is not an issue. maintainability is.